### PR TITLE
Removes Edge support.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: modules/RTC/ortc/RTCPeerConnection.js [hunk 1]
```diff
@@ -1,2146 +0,0 @@
-/* global __filename, RTCIceGatherer, RTCIceTransport, RTCDtlsTransport,
-RTCRtpSender, RTCRtpReceiver */
-
-import { getLogger } from 'jitsi-meet-logger';
-import yaeti from 'yaeti';
```
Comment (saghul) [inline modules/RTC/ortc/RTCPeerConnection.js]: you can also remove this dependency from package.json

Comment (damencho) [reply modules/RTC/ortc/RTCPeerConnection.js]: Removed it.

**Comment by dennishansen** [comment]
Why did we remove edge support? If we add it back, should we have any issues?

**Comment by damencho** [comment]
This removes edge support for the non chromium version, which anyway had its problems and soon will no longer exist.

**Pull request merged**